File: Exercicios-Beecrowd/C#/1010.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Runtime.Serialization;
using System.Text;
using System.Threading.Tasks;

namespace Teste
{
    internal class Program
    {
        static void Main(string[] args)
        {
            string[] lista_1= Console.ReadLine().Split();
            string[] lista_2 = Console.ReadLine().Split();
            double a = double.Parse(lista_1[1]);
            double b = double.Parse(lista_1[2]);
            double c = double.Parse(lista_2[1]);
            double d = double.Parse(lista_2[2]);
            double soma_1 = a * b;
            double soma_2 = c* d;
            double  resultado = soma_1 + soma_2;
            string final = resultado.ToString("F2");
            Console.WriteLine("VALOR A PAGAR: R$ {0}",final);

        }
    }
}

File: Exercicios-Beecrowd/Kotlin/1010.kt
import java.util.*

fun main(args: Array<String>) {

	val peca1= readLine().toString().split(" ")
    val peca2= readLine().toString().split(" ")
    val valor1=peca1[1].toInt()
    val valor2=peca1[2].toDouble()
    val valor3=peca2[1].toInt()
    val valor4=peca2[2].toDouble()
    val resultado= (valor1*valor2)+(valor3*valor4)
    val formatado= ("%.2f".format(resultado))
    println("VALOR A PAGAR: R$ ${formatado}")
	
}